code/game/machinery/defibcharger.dm

```diff
@@ -87,11 +87,10 @@ obj/machinery/recharger/defibcharger/wallcharger/process()
 	return -1
 
 obj/machinery/recharger/defibcharger/wallcharger/attackby(obj/item/weapon/G as obj, mob/user as mob)
-	if(..())
-		return
 	if(istype(user,/mob/living/silicon))
 		return
 	if(istype(G, /obj/item/weapon/melee/defibrillator))
+		..()
```

Comment:
..() by itself calls the parent to check if it's powered or not, it returns when it's not.

With the above code it won't even bother to call the parent if it's not a defib, which makes it so that I don't even have to edit the parent for the behavior I want.